src/app/components/Hyperlink/index.tsx

```diff
@@ -1,14 +1,26 @@
 import { ReactNode } from "react";
+import { classNames } from "~/app/utils";
 
 type Props = {
-  onClick: () => void;
+  href?: string;
+  onClick?: () => void;
   children: ReactNode;
+  className?: string;
 };
 
-export default function Hyperlink({ onClick, children }: Props) {
+export default function Hyperlink({
+  onClick,
+  children,
+  href,
+  className,
+}: Props) {
   return (
     <a
-      className="cursor-pointer text-sky-500 hover:text-sky-400"
+      className={classNames(
+        "cursor-pointer text-blue-700 hover:text-blue-500",
+        className ?? ""
```

Comment:
can't you just do `className`? if it is falsy it should be omitted, right?

Reply:
It's an optional param for the component that isn't compatible with the classNames() func, thus the check.

> Argument of type 'string | undefined' is not assignable to parameter of type 'string | boolean'.